integrated_channels/integrated_channel/learner_data.py

```diff
@@ -25,26 +25,30 @@ class BaseLearnerExporter(object):
     Base class for exporting learner completion data to integrated channels.
     """
 
+    GRADE_PASSING = 'Pass'
+    GRADE_FAILING = 'Fail'
+    GRADE_INCOMPLETE = 'In Progress'
+
     @property
     def grade_passing(self):
         """
         Returns the string used for a passing grade.
         """
-        return 'Pass'
+        return self.GRADE_PASSING
```

Comment:
I'm not fussed about whether this should be a property or a constant.  It's possible that other integration channels will want something totally different, e.g. boolean pass/fail, or numeric grade, or who knows what, so it's hard to know how best to abstract it with just one example.